base/kilda-base-ubuntu/Dockerfile

```diff
@@ -13,24 +13,24 @@
 #   limitations under the License.
 #
 
-FROM ubuntu:xenial
+FROM ubuntu:18.04
```

Comment:
Is it necessary to use 18.04 instead of 16.04 ?
I think we should leave xenial distrib as a base for now